src/browserExt/contentTypeHandler.js

```diff
@@ -147,13 +165,13 @@ Zotero.MIMETypeHandler = {
 						Zotero.Connector.callMethod(options, this.response, function(result, status, response) {
 							if (result) {
 								Zotero.Messaging.sendMessage('progressWindow.itemProgress',
-									[Zotero.ItemTypes.getImageSrc('dictionaryEntry'), result.name, null, 100], tab);
+									[chrome.extension.getURL('images/csl-style.png'), result.name, null, 100], tab);
```

Comment:
I'll address this more thoroughly with a separate commit, but I've changed this icon to be 2x and downscaled. Currently the connector icons are a bit of a mess. Some are 2x, some are not, no HiDPI detection at all.